src/splitInteger.test.js

```diff
@@ -8,12 +8,27 @@ describe(`Function 'splitInteger':`, () => {
   });
 
   it(`should return an array`, () => {
-
+    expect(splitInteger()).toBeInstanceOf(Array);
   });
 
   it(`should return an array with 'num' if parts = 1`, () => {
+    expect(splitInteger(8, 1)).toEqual([8]);
+  });
```

Comment:
How about adding a test where 'num' is divisible by 'parts'?

Reply:
> How about adding a test where 'num' is divisible by 'parts'?

Added